### DISCUSSION TITLE
One of the locales messages not loading with lazy load

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt-modules/i18n#2284] [Q&A] [answered]
I've been using the lib since the v8 beta. It has been working great with `lazy: true`, and my locales JSON files.
 
I have a scaffolding tool to create the nuxt app templates, so each time the configuration is the same.
It also updates all deps, and it installed latest version `8.0.0.-rc.1`.

Today I decided trying it to help someone who had a different issue.

And I noticed the messages from `lang/es-ES.json` are not loading and I get this error in the browser: 
`core-base.mjs?v=b250aea8:792 [intlify] Not found 'message' key in 'es-ES' locale messages.`

So I started to play with the config in `nuxt.config.ts`, changed `strategy` around, `detectBrowserLanguage: false`.
Nothing.

Then I decided setting `lazy: false` and bam, it worked right away!

So I'm wondering now if the lazy loading logic is failing in this version, or if I have to "force it" to load now. Checked the methods in `useI18n()` but couldn't find anything.

It works under 2 circumstances:

1. If I set `lazy: false`.
2. If I set `defaultLocale` to `es-ES`.

### I18n Nuxt Config

```ts
i18n: {
    langDir: 'lang',
    lazy: true,
    defaultLocale: 'en-US',
    
    locales: [
      {
        code: 'en-US',
        name: 'English (United States)',
        file: 'en-US.json',
      },
      {
        code: 'es-ES',
        name: 'Español (España)',
        file: 'es-ES.json',
        
      }
    ]
  }
```

### Locale Files

`lang/en-US.json`

```json
{
  "message": "Hello Locale"
}
```
`lang/es-ES.json`

```json
{
  "message": "Hola Localización"
}
```

### Environment
OS: Windows 11 Home latest
nuxt: 3.6.5
@nuxtjs/i18n: 8.0.0.-rc.1

### Final Words

Any information will be appreciated.

**Comment by angelhdzmultimedia** [comment]
### Solved?

Apparently the solution was to  set `fallbackLocale: ['en-US', 'es-ES']` in `i18n.config.ts`.

Makes no sense because I'm already defining the locales by code and filename...

![image](https://github.com/nuxt-modules/i18n/assets/10191085/5f043857-5c4c-420b-bb17-1b0c8c264d7d)

But my `es-ES` locale isn't missing!

**Reply by kazupon** [reply, marked as answer]
related issue https://github.com/nuxt-modules/i18n/issues/2192

**Reply by angelhdzmultimedia** [reply]
I just tried with latest version of all packages, and the issue is gone, I could remove the i18n.config.ts file, and all the locales (and the default one) worked, so I'm going to close my discussion. Thank you to all the team!

**Comment by angelhdzmultimedia** [comment]
### Solved
